src/power_divergence.jl

```diff
@@ -119,31 +120,24 @@ function ci_sison_glaz(x::PowerDivergenceTest, alpha::Float64; skew_correct::Boo
             lambda = x.observed[i]
             #run moments
             a = lambda + c
-            b = max(lambda - c, 0)
-            if lambda > 0
-                poislama = cdf(Poisson(lambda), a)
-                poislamb = cdf(Poisson(lambda), b - 1)
-            else
-                poislama = poislamb = 1.0
-            end
-            den = b > 0 ? poislama-poislamb : poislama
+            b = max(lambda - c, 0.0)
```

Comment:
Please don't change these literals. There are a few more below.

Reply:
Sorry can you explain a bit more?

Reply:
The original version uses `0` which is just fine here and more generic than the `Float64` literal which might be useful later.